ftsbot/cogs/textcommands.py

```diff
@@ -547,17 +570,39 @@ async def on_thinking_error(
 	@app_commands.command(
 		description='Help on how to submit a photo'
 	)
+	@app_commands.describe(
+		user='Which user is the guidelines directed towards?'
+	)
 	async def photos(
 		self,
-		interaction: discord.Interaction
+		interaction: discord.Interaction,
+		user: typing.Optional[discord.User]
 	):
+		if user is None:
+			await interaction.response.send_message(
+				embed=discord.Embed(
+					colour=discord.Colour(0x00ffff),
+					description=(
+						'Please have the copyright owner email the image to "photos@liquipedia.net", '
+						+ 'alongside a statement giving their permission for it to be used on Liquipedia. '
+						+ 'Please also include what wiki the image is for and the player in question.'
+					)
+				)
+			)
 		await interaction.response.send_message(
```

Comment:
The second block would need to be an `else:` or there would be two messages I think.